.github/workflows/core.yml

```diff
@@ -30,33 +30,29 @@ jobs:
           cache: 'yarn'
       - run: yarn install --immutable --immutable-cache
       - run: yarn docs:build
-  create-script-with-yarn:
-    strategy:
-      fail-fast: false
-      matrix:
-        integration: [magento, shopware, px]
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/setup-node@v3
-        with:
-          node-version: 18
-      - run: |
-          yarn create storefront-x --directory=app --integration=${{ matrix.integration }}
-          cd app
-          yarn install
-          yarn build
-  create-script-with-npm:
+  create-script-with-npm-locale:
     strategy:
       fail-fast: false
       matrix:
         integration: [magento, shopware, px]
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
           node-version: 18
       - run: |
-          npm create storefront-x@latest -y -- --directory=app --integration=${{ matrix.integration }}
-          cd app
+          npm install
+          cd modules/create-storefront-x
+          npm install
+          node index.js -y -- --directory=../../app --integration=${{ matrix.integration }}
+          cd ../../app
+          for dep in $(grep -Eo '"@storefront-x\/[a-zA-Z0-9-]+"' package.json | sed -E 's/[^a-zA-Z0-9-]//g' | sed 's/storefront-x//g'); do
```

Comment:
Dont iterate over modules in pacakge.json. Iterate over directories inside storefront-x/modules